pygmt/enums.py

```diff
@@ -0,0 +1,39 @@
+"""
+Enumerations for PyGMT.
+"""
+
+from enum import IntEnum
+
+
+class GridFormat(IntEnum):
+    """
+    Enum for the GMT grid format ID.
+
+    These enums are defined in 'gmt_grdio.h'.
+    """
+
+    UNKNOWN = 0  #: Unknown grid format
+    BF = 1  #: GMT native, C-binary format (32-bit float)
+    BS = 2  #: GMT native, C-binary format (16-bit integer)
+    RB = 3  #: SUN rasterfile format (8-bit standard)
+    BB = 4  #: GMT native, C-binary format (8-bit integer)
+    BM = 5  #: GMT native, C-binary format (bit-mask)
+    SF = 6  #: Golden Software Surfer format 6 (32-bit float)
+    CB = 7  #: GMT netCDF format (8-bit integer, depreciated)
+    CS = 8  #: GMT netCDF format (16-bit integer, depreciated)
+    CI = 9  #: GMT netCDF format (32-bit integer, depreciated)
+    CF = 10  #: GMT netCDF format (32-bit float, depreciated)
+    CD = 11  #: GMT netCDF format (64-bit float, depreciated)
```

Comment:
Deprecated?
```suggestion
    CB = 7  #: GMT netCDF format (8-bit integer, deprecated)
    CS = 8  #: GMT netCDF format (16-bit integer, deprecated)
    CI = 9  #: GMT netCDF format (32-bit integer, deprecated)
    CF = 10  #: GMT netCDF format (32-bit float, deprecated)
    CD = 11  #: GMT netCDF format (64-bit float, deprecated)
```

Reply:
The word "depreciated" comes from https://docs.generic-mapping-tools.org/dev/reference/features.html#grid-file-format-specifications. Reading https://stackoverflow.com/a/9208164, I think "depreciated" is still correct.

Reply:
Depreciated is a monetary term (comes from the word depreciation). I still think it's a typo.

Reply:
Makes sense. Applied the suggestions.